src/main/msp/msp.c

```diff
@@ -1469,6 +1469,9 @@ case MSP_NAME:
 #ifdef USE_MAG
     case MSP_COMPASS_CONFIG:
         sbufWriteU16(dst, imuConfig()->mag_declination);
+        sbufWriteU16(dst, compassConfig()->mag_customAlignment.roll);
```

Comment:
IMO this should be handled as sensor alignment, common for all sensors

Reply:
Moved to MSP_SENSOR_ALIGNMENT

Reply:
Is the else block in https://github.com/betaflight/betaflight/pull/14019/files#diff-5dc4eb2b2548bfc728653a89570779dabe03e47e60f72ad64d36c5d6c31eb5f6R3008-R3023 still valid ?